plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/ConnectionFactory.java

```diff
@@ -81,13 +83,28 @@ public boolean hasDefaultHost() {
 
 	@Override
 	public String getUserDocUrl() {
+//		return getUserDocumentationUrl();
+		return getSignupUrl();
+	}
+
+	@SuppressWarnings("unused")
+	private String getUserDocumentationUrl() {
```

Comment:
currently we dont. I was unsure whether we should still show it but changing this would require a larger change which I decided not to go for. Removing it to match "dont keep dead code".